R/workbook_write_data.R

```diff
@@ -163,15 +172,31 @@ Workbook$methods(writeData = function(df, sheet, startRow, startCol, colNames, c
   }
 
 
+  ref_cell <- paste0(int_2_cell_ref(startCol), startRow)
 
-  ## Forumlas
+  ## Formulas
   f_in <- rep.int(as.character(NA), length(t))
   any_functions <- FALSE
+
   if ("openxlsx_formula" %in% colClasses) {
 
     ## alter the elements of t where we have a formula to be "str"
     formula_cols <- which(sapply(colClasses, function(x) "openxlsx_formula" %in% x, USE.NAMES = FALSE), useNames = FALSE)
-    formula_strs <- stri_join("<f>", unlist(df[formula_cols], use.names = FALSE), "</f>")
+    formula_strs <- stri_join("<f aca=\"false\">", unlist(df[formula_cols], use.names = FALSE), "</f>")
```

Comment:
no clue what this aca="false" statement does. have seen it with libreoffice files. presumably it can be dropped again